Tests/Unit/Command/FrontendCommandControllerTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace  Helhum\Typo3Console\Tests\Unit\Command;
+
+use Helhum\Typo3Console\Command\FrontendCommandController;
+use Nimut\TestingFramework\TestCase\UnitTestCase;
+
+class FrontendCommandControllerTest extends UnitTestCase
+{
+    /**
+     * Call protected/private method of a class.
+     *
+     * @param object &$object    Instantiated object that we will run method on.
+     * @param string $methodName Method name to call
+     * @param array  $parameters Array of parameters to pass into method.
+     *
+     * @return mixed Method return.
+     */
+    public function invokeMethod(&$object, $methodName, array $parameters = [])
+    {
+        $reflection = new \ReflectionClass(get_class($object));
+        $method = $reflection->getMethod($methodName);
+        $method->setAccessible(true);
+
+        return $method->invokeArgs($object, $parameters);
+    }
+
+    /**
+     * @test
+     * @dataProvider makeAbsoluteDataProvider
+     * @param mixed $expected
+     * @param mixed $given
+     */
+    public function makeAbsoluteTest($expected, $given)
+    {
+        /** @var FrontendCommandController $frontendRequest */
+        $frontendRequest = new FrontendCommandController();
+        $parameter = [ $given ];
+        $this->assertEquals($expected, $this->invokeMethod($frontendRequest, 'makeAbsolute', $parameter));
+    }
+
+    /**
+     * @return array
+     */
+    public function makeAbsoluteDataProvider()
+    {
+        return [
+            'Relative path to home' => ['http://localhost/', '/'],
+            'Relative path to foo without trailing slash' => ['http://localhost/foo', '/foo'],
+            'Relative path to foo with trailing slash' => ['http://localhost/foo/', '/foo/'],
+            'Test with HTTP scheme' => ['http://typo3.org/', 'http://typo3.org/'],
+            'Test with HTTPS scheme' => ['https://typo3.org/', 'https://typo3.org/'],
+            'Test with open scheme' => ['//typo3.org/', '//typo3.org/'],
```

Comment:
I don't think it makes sense to allow that. Not sure what side effects could occur. Why do you think it is needed?